src/plugins/imagecapturer/CImageCapturer.cpp

```diff
@@ -114,6 +119,10 @@ namespace semrec {
       try {
 	if(sensor_msgs::image_encodings::isColor(m_imgReceived.encoding)) {
 	  cv_ptr = cv_bridge::toCvCopy(m_imgReceived, sensor_msgs::image_encodings::BGR8);
+  } else if(isDepthImage(m_imgReceived.encoding)) {
```

Comment:
In which direction should I fix it? Spaces and tabs are mixed all over the file.